src/System.Security.Principal.Windows/src/System/Security/Principal/NTAccount.cs

```diff
@@ -371,8 +371,6 @@ private static IdentityReferenceCollection TranslateToSids(IdentityReferenceColl
             finally
             {
                 LsaHandle.Dispose();
-                ReferencedDomainsPtr.Dispose();
```

Comment:
We do want to call Dispose here if there was actual handle allocated by the code above.

Reply:
added check if handle is valid...if I understood well